fvm/fvm.go

```diff
@@ -56,8 +56,8 @@ func (vm *VirtualMachine) Run(ctx Context, proc Procedure, v state.View, program
 			// Cadence may fail to encode certain values.
 			// Return an error for now, which will cause transactions to revert.
 			//
-			if encodingErr, ok := r.(interpreter.EncodingUnsupportedValueError); ok {
-				err = errors.NewEncodingUnsupportedValueError(encodingErr.Value, encodingErr.Path)
+			if _, ok := r.(interpreter.UnsupportedTagDecodingError); ok {
+				//err = errors.NewEncodingUnsupportedValueError(encodingErr.Value, encodingErr.Path)
```

Comment:
not sure why we commented this out, @janezpodhostnik is this safe ?

Reply:
Reverting this change

Reply:
EncodingUnsupportedValueError was actually removed from cadence, so this whole part of the code can be removed, But that should probably be a different PR (like this one: https://github.com/onflow/flow-go/pull/1975)